app/build.gradle.kts

```diff
@@ -107,6 +108,8 @@ dependencies {
 
     implementation(platform(libs.google.firebase.bom))
     implementation(libs.google.firebase.messaging)
+    implementation(libs.google.firebase.analytics)
```

Comment:
Should there be analytics in this PR?

Reply:
Yes, We should add analytics 
![image](https://github.com/SoftTeco/AndroidAppTemplate/assets/76498024/d06c9941-3f1f-4c43-86e9-da9ff2bfe9ad)